From a6e28a8414a056104f740fe0a8ea92fd542f3374 Mon Sep 17 00:00:00 2001
From: "qiang.cheng" <qiang.cheng@amlogic.com>
Date: Mon, 6 Nov 2023 18:33:37 +0800
Subject: [PATCH 02/13] debian: revert
 'I6509c3be68e8c313f321f47e14f03265920bda16' [1/1]

PD#SWPL-144374

Problem:
revert 'I6509c3be68e8c313f321f47e14f03265920bda16'

Solution:
revert 'I6509c3be68e8c313f321f47e14f03265920bda16'

Verify:
local

Signed-off-by: qiang.cheng <qiang.cheng@amlogic.com>
Change-Id: I1dc33af0b6bfb606196d1adb0fd37c8781fffecf
---
 env/copy_debs_from_yocto.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/env/copy_debs_from_yocto.sh b/env/copy_debs_from_yocto.sh
index 76cd36f..1e11ca9 100755
--- a/env/copy_debs_from_yocto.sh
+++ b/env/copy_debs_from_yocto.sh
@@ -88,7 +88,7 @@ package_status_append=(kernel-5.15.119_5.15.119+git999-r2_arm64.deb \
 			python3-dbus_1.2.16-r0_arm64.deb \
 			python3-pygobject_3.34.0-r0_arm64.deb \
 			sqlite3_3.31.1-r0_arm64.deb \
-			wayland_1.19.0-r0_arm64.deb \
+			wayland_1.17.0-r0_arm64.deb \
 			gstreamer1.0_1.16.3-r0_arm64.deb \
 			libkms1_2.4.100-r0_arm64.deb \
 			wpa-supplicant-passphrase_2.9-r0_arm64.deb \
-- 
2.25.1

